packages/ui/src/kit/VTooltip/VTooltip.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <transition name="v-tooltip-fade">
+    <div
+      v-show="show"
+      class="v-tooltip"
+      :class="tagCssClass"
+      v-bind="$attrs"
+    >
+      <div class="v-tooltip-container">
+        <div class="v-tooltip-triangle" />
+        <slot />
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import ThemeMixin from '@/mixins/ThemeMixin';
+
+export default {
+  name: 'VTooltip',
+
+  inheritAttrs: false,
+
+  props: {
+    show: {
+      type: Boolean,
+      default: false,
+    },
+    position: {
+      type: String,
+      default: 'top-left',
+      validator(value) {
+        return (
+          ['top-left', 'top-right', 'bottom-left', 'bottom-right'].indexOf(
```

Comment:
I think tooltip should have `top`, `left` etc position variants, not current. It is common behavior for tooltip.

Reply:
don't think soo, if we need center positions in top or bottom - it will be not a certain story =\

Reply:
just like `top-center` or `bottom-center`